src/Phinx/Migration/Migration.template.php.dist

```diff
@@ -3,6 +3,13 @@ $namespaceDefinition
 
 use $useClassName;
 
+/**
+ * $className migration
+ *
+ * @deprecated up() This method is deprecated and will be removed in a future release. Use change($direction).
```

Comment:
Does this work? Or will IDEs show all class usage as deprecated? Then we better use an inline deprecation:

    * Deprecated up() ...

Reply:
Aha! Yes. The intent is to show only `up()` and `down()` are deprecated, not the migration itself.

I wonder if just passing `$direction` to `change()` would be enough first. Add a deprecation notice of `up()` and `down()` to the release notes and a new upgrade doc (next release) that documents a what's required to convert existing migrations from `up()`/`down()` to `change($direction)`, though it is pretty obvious.